tests/driver_lc709203f/main.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2017 RWTH Aachen, Josua Arndt, Steffen Robertz
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       test aplication for the LC709203F Battery fuel gauge by on semiconductors
+ *
+ * @author	Steffen Robertz <steffen.robertz@rwth-aachen.de>
+ * @author	Josua Arndt <jarndt@ias.rwth-aachen.de>
+ * @}
+ */
+
+#include "board.h"
+#include "periph_conf.h"
+#include "lc709203f.h"
+#include "xtimer.h"
+
+#ifndef LC709203F_INT_PIN
+    #define LC709203F_INT_PIN GPIO_PIN(PORT_E, 6)
+#endif
+
+static void _gauge_cb(void *arg)
+{
+    printf("\n ALARM: Low RSOC \n");
+}
+
+int main(void)
+{
+    puts("LC709203F Fuel Gauge test application");
+    lc709203f_init(I2C_0, LC709203F_INT_PIN, _gauge_cb, NULL);
+
+    puts("The application will trigger an low_rsoc interrupt in 2s");
+    xtimer_sleep(2);
+    lc709203f_set_alarm_low_rsoc(I2C_0, 0);
+    lc709203f_set_alarm_low_rsoc(I2C_0, 100);
+    puts("Read low rsoc threshold level. Should be 100");
+    printf("Alarm Low RSOC level: %u \n", lc709203f_get_alarm_low_rsoc(I2C_0));
```

Comment:
does it make sence to insert a test for the other get/set methods?

Reply:
@MichelRottleuthner I don't think it would make sense. All get/set methods are more or less copys of each other with a different register set. If one is working, all other ones should be fine too.
The other ones should also be less important.

Reply:
Actually I didn't think of it as a test for the driver logic but more as a nice to have feature for testing a real device in a "well defined" test environment where you compare the tests output with the expected values.  But Looking at the readable data again, it doesn't seem like any of them - apart from the rsoc values - are directly influenced by signals measured by the ic. So I agree with you, leave it as is.